### ISSUE TITLE
Get current cursor shape doesn't work with HTML5 export

### ISSUE THREAD

**Issue by sygi** [godotengine/godot#66835]
### Godot version

3.5-stable

### System information

Ubuntu 22.04

### Issue description

Input class has methods for changing the cursor and [checking the current cursor shape](https://docs.godotengine.org/en/stable/classes/class_input.html#class-input-method-get-current-cursor-shape). They work fine with native build, but when exporting to HTML5, the latter method always responds with the default cursor shape (0), whether a custom one is currently set or not.

### Steps to reproduce

Export to HTML5 a project where a root node is:

```
func _ready():
  Input.set_current_cursor_shape(6)
  print(Input.get_current_cursor_shape())  # prints 0
```

**Comment by akien-mga** [comment]
That code snippet doesn't compile, there's no `Input.set_current_cursor_shape()` method. I guess you used `set_custom_mouse_cursor()` instead? But if so we need a clear example of how you're calling that method.

**Comment by akien-mga** [comment]
I can reproduce the issue, it seems like `get_cursor_shape` is not implemented in `OS_JavaScript`.

**Comment by akien-mga** [comment]
Fixed by #66871.
